src/lib.rs

```diff
@@ -145,11 +146,19 @@ impl<'a> Image<'a> {
 						});
 					}
 					ColorMode::Rgb => {
-						cell.set_char(BLOCK_FULL).set_fg(Color::Rgb(
-							(255.0 * r) as u8,
-							(255.0 * g) as u8,
-							(255.0 * b) as u8,
-						));
+						if false {
```

Comment:
i guess this is a work in progress then?

Reply:
Just fixed that, should be fully working with RGB now.